lib/dehinter/font.py

```diff
@@ -14,12 +14,121 @@
 
 import array
 import os
+import pprint
+import sys
 from typing import Union
 
 from fontTools import ttLib  # type: ignore
 
 from dehinter.bitops import clear_bit_k, is_bit_k_set
 
+# instantiate pretty printer
+pp = pprint.PrettyPrinter(indent=4)
+
+
+# ========================================================
+# Core dehinting routine
+# ========================================================
+def dehint(tt,
+           keep_cvar=False,
+           keep_cvt=False,
+           keep_fpgm=False,
+           keep_gasp=False,
+           keep_glyf=False,
+           keep_hdmx=False,
+           keep_head=False,
+           keep_ltsh=False,
+           keep_maxp=False,
+           keep_prep=False,
+           keep_ttfa=False,
+           keep_vdmx=False,
+           verbose=True):
+    def report_actions(table, has_table):
```

Comment:
Will you please pull this function out to the module level?

Reply:
the reason why I placed it as an inner-function was to not expose it to the public API. This is only ever used as a helper function in the implementation of this specific function, so I considered it not ideal to make it visible outside of this function scope.

Reply:
I can address this issue.  Let's go ahead and merge this so that you can prep your fontbakery PR with the changes. This isn't a huge deal.  It is a just a style preference on my end.